en/docs/services/create-your-first-service.md

```diff
@@ -173,30 +173,29 @@ To test the `CovidStatus` service you created, follow the procedure below:
 
 4. Click **Try it out**, and in the **country** field, enter `USA`.
    
-5. Click **Execute**.
-
-    The response is displayed as follows:
+5. Click **Execute**. The response is displayed as follows:
 
     ![Response in the test view](../assets/img/services/service-response.png){.cInlineImage-half}
 
 
 ## Step 7: Deploy the service
 
-To deploy the `CovidStatus` service, follow the procedure below:
+To deploy the `CovidStatus` service, follow this procedure:
 
 1. Click the **Go Live** icon in the left pane.
 
     ![Deploy icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
 
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs appear to indicate that the service is successfully deployed.
-    Then, click on **://cURL**. Copy the cURL command that is displayed.
+    The status of the service changes to **Deployed**, and the following logs appear, indicating that the service is successfully deployed.
 
     ![Get cURL command](../assets/img/services/get-curl-command.png){.cInlineImage-full}
     
+    Then click **://cURL** and copy the cURL command that is displayed.
+    
     !!! tip
-        The cURL command will be something as given in the example below:<br/><br/>
+        The following shows an example of what the cURL command might look like:<br/><br/>
```

Comment:
might look like?

Reply:
Agree that the term "might" in the copy editor's suggestion doesn't seem right. 
I changed it to "would look like". WDYT?